docs/dev/build_riscv64.md

```diff
@@ -69,7 +69,9 @@ The software was validated on the following devices:
      -DRISCV_TOOLCHAIN_ROOT=/opt/riscv
    ```
    > **NOTE**: The `riscv-gnu-toolchain` is build as there are essential files used for cross compilation under `/opt/riscv/sysroot`. The latest stable versions of Clang or GCC both support compiling source code into RISC-V instructions, so it is acceptable to choose your preferable compilers by specifying `-DCMAKE_C_COMPILER` and `CMAKE_CXX_COMPILER`. But remember to add the key `-DCMAKE_SYSROOT=/opt/riscv/sysroot`, otherwise many fundamental headers and libs could not be found during cross compilation. 
-   
+
+   By default OpenVINO is built without parallelism support on RISC-V devices. However there is OpenMP support which can be enabled using the cmake key `-DTHREADING=OMP`.
```

Comment:
Maybe we should set it to ON by default in cmake for RISC-V ?
I hope it will expose better performance out of the box than SEQ version

Reply:
Add commit https://github.com/openvinotoolkit/openvino/pull/24352/commits/27082610238f323859a683ad78debc2e492e8719. Let's take a look at GHA riscv64 job. I hope it will be passes 😃